tests/src/ExportMap.js

```diff
@@ -0,0 +1,21 @@
+import { expect } from 'chai';
+import { readTsConfig } from '../../src/ExportMap';
```

Comment:
let's please not export things solely to test them; can these test fixtures be used to text the actual *effects* of the bug, as reported in the original issue?

Reply:
I'll make that change now :)